symbolic-demangle/src/lib.rs

```diff
@@ -183,19 +214,16 @@ fn try_demangle_swift(ident: &str, opts: DemangleOptions) -> Option<String> {
         Err(_) => return None,
     };
 
-    let simplified = match opts.format {
-        DemangleFormat::Short => {
-            if opts.with_arguments {
-                1
-            } else {
-                2
-            }
-        }
-        DemangleFormat::Full => 0,
-    };
+    let mut features = 0;
+    if opts.return_type {
+        features |= 1 << 1;
```

Comment:
Can we introduce shared constants for these to make this searchable across C++/Rust code?